examples/advanced-pytorch/server.py

```diff
@@ -10,8 +10,6 @@
 
 import warnings
 
-from flwr_datasets import FederatedDataset
-
```

Comment:
This should be moved to another PR ideally

Reply:
Sure, will remove it :)